features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/SessionReplayFeature.kt

```diff
@@ -211,6 +211,17 @@ internal class SessionReplayFeature constructor(
     // endregion
 
     internal companion object {
+
+        /**
+         * Session Replay storage configuration with the following parameters:
+         * * max item size = 4 MB,
+         * * max items per batch = 500,
+         * * max batch size = 10 MB, SR intake batch limit is 10MB
+         * * old batch threshold = 18 hours.
+         */
+        internal val STORAGE_CONFIGURATION: FeatureStorageConfiguration =
+            FeatureStorageConfiguration.DEFAULT.copy(maxBatchSize = 10 * 1024 * 1024)
```

Comment:
Shouldn't the maxItemSize be increased as well?

Reply:
I am actually assessing this...I think 512KB should be enough...let me check with Maciej

Reply:
Yes, you were right, they increased that too so right now we are 10MB, 10MB for both item and bath size to align with them for SR